picking_good_slos.md

```diff
@@ -0,0 +1,55 @@
+# How to pick a good SLO
+
+A Service Level Objective (SLO) is a measurable, specific goal for a given service's reliability level of service over time, based on one or more SLIs.
+To learn more about SLIs, check out our article on how to [pick good SLIs](./picking_good_slis.md).
+
+An SLO is used to quantify customer experience.
+Furthermore, SLOs enable us to avoid false positives in alerting and help us focus on symptom-based alerting.
+SLOs can vary from organization to organization. Consequently, there is no single formula that we could use to define the appropriate reliability target. 
+In this article we will share some tips for picking a good SLO.
+
+So, what should we consider when picking the SLO for a service?
+
+It's important to understand the outcomes of setting an SLO. These outcomes differ depending on multiple factors, including both the maturity of the service and the whole organization overall.
+To understand the different SLO settings, let's explore some use-cases that appear at either end of SLO adoption.
+
+## Case 1 - New Service
+
+For this case, imagine a mature SRE organization. In this organization, we have identified stakeholders spanning SLO [personas](./ADRs/RH/SIG-SRE/ADR-00002%20Personas%20related%20to%20Managed%20Services.md).
+Each stakeholder understands the outcomes of both meeting or missing an SLO. 
+
+So with that in mind, what should be our service level objective? 
+
+The fact that this case has a new service brings forward the opportunity to use an SLO to steer development efforts before reaching production. 
```

Comment:
s/The fact that this case has a new service brings forward/A new service presents